stable/grafana/templates/deployment.yaml

```diff
@@ -59,6 +59,15 @@ spec:
               readOnly: {{ .readOnly }}
           {{- end }}
 {{- end }}
+        - name: set-permissions
+          image: busybox
+          command: ["/bin/sh", "-c", "mkdir -m 777 /var/lib/grafana/plugins && chmod -R 777 /var/lib/grafana/"]
```

Comment:
Should we run chown 472:472 instead of chmod 777?

Reply:
yes, you're right, no need to set 777, just changed that, thanks